docs/sphinx/dev_guide/ci_tasks.rst

```diff
@@ -157,6 +157,113 @@ annotate the job for this. For example:
    describe the change in the ``RAJA/.gitlab/jobs/<MACHINE>.yml`` file where
    the job is overridden.
 
+
+Building the Intel clang + SYCL HIP compiler for use in CI
+------------------------------------------------------
+
+The SYCL CI tests on corona rely on a custom Intel Clang SYCL compiler that we 
+build ourselves. This compiler lives in the /usr/workspace/raja-dev/ folder so 
+that it can be accessed by the gitlab CI system. Since the intel compiler does
+not do releases in the typical sense (they simply update their repo *every night*), 
+it may become necessary to periodically build a new version of the compiler to 
+ensure that we are using the most up-to-date version available. The steps for 
+building/installing/running are shown here.
+
+Building the Compiler
+^^^^^^^^^^^^^^^^^^^^^
+
+.. important:: Because intel update's their compiler repo daily, there is a nonzero possibility that the had of the sycl branch will build. 
```

Comment:
```suggestion
.. important:: Because intel updates their compiler repo daily, there is a nonzero possibility that the had of the sycl branch will build. 
```